src/models/epochs.rs

```diff
@@ -5,8 +5,8 @@ use serde::Deserialize;
 pub struct Epoch {
     pub blk_count: i32,
     pub epoch_no: i32,
-    pub fees: i32,
-    pub start_time: i32,
+    pub fees: String,
+    pub start_time: i64,
```

Comment:
The `fees` were coming back as a `String` which was causing any request that used this struct to fail deserialization.